src/posix/main.c

```diff
@@ -150,7 +148,6 @@ static void PrintUsage(const char *aProgramName, FILE *aStream, int aExitCode)
             "        --ncp-dataset             Retrieve and save NCP dataset to file.\n"
             "    -s  --time-speed factor       Time speed up factor.\n"
             "    -v  --verbose                 Also log to stderr.\n"
-#if OPENTHREAD_POSIX_RCP_SPI_ENABLE
```

Comment:
If the macro "OPENTHREAD_POSIX_RCP_SPI_ENABLE" is removed,  posix-host will output SPI help information regardless of whether SPI interface is enabled. Could we print SPI and HDLC information separately, and then use macros to control which information is printed?